frontend/integration-tests/tests/kubevirt/vm.yaml.scenario.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+import { browser, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { removeLeakedResources } from './utils';
+import { detailViewAction, detailViewVmIcon, runningIcon } from '../../views/kubevirt/vm.actions.view';
+import Yaml from './models/yaml';
+
+
+describe('Test create vm from yaml', () => {
+  const leakedResources = new Set<string>();
+  const yaml = new Yaml();
+
+  beforeEach(async() => {
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+    await isLoaded();
+    await yaml.openYamlPage();
+  });
+
+  afterAll(async() => {
+    removeLeakedResources(leakedResources);
+  });
+
+  describe('Test create VM from the default yaml file', () => {
+    it('Creates VM', async() => {
+      await yaml.createVMFromYaml();
+      await isLoaded();
+      await detailViewAction('Start');
+      await browser.wait(until.presenceOf(detailViewVmIcon(runningIcon)));
+    });
+  });
+
+  describe('Test create VM from the default yaml file again', () => {
```

Comment:
probably no big deal, but I don't think it is a good practice to depend on the order of the tests.

Reply:
the original purpose is emulating a failure scenario, and yeah, it's not big.